core/tests/ts-integration/tests/custom-account.test.ts

```diff
@@ -40,7 +45,18 @@ describe('Tests for the custom account behavior', () => {
 
     test('Should deploy custom account', async () => {
         const violateRules = false;
-        customAccount = await deployContract(alice, contracts.customAccount, [violateRules], 'createAccount');
+        customAccount = await deployContract(
+            alice,
+            contracts.customAccount,
+            // 2555971200 is a number of seconds up to 30/12/2050
```

Comment:
Nit: This comment should be moved to the `DEFAULT_TIMESTAMP_ASSERTER_RANGE_END` declaration.